unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/KernelCommunication/WebSocketCommunication/WebSocketCommunication.cs

```diff
@@ -25,7 +25,7 @@ public class WebSocketCommunication : IKernelCommunication
 
     WebSocketServer ws;
 
-    public WebSocketCommunication(bool withSSL = false, int startPort = 7666, int endPort = 7800)
+    public WebSocketCommunication(bool withSSL = false, int startPort = 5000, int endPort = 7800)
```

Comment:
can we extract this magic number in a public cosnt and then re-use it in `Main` and `MainDesktop` classes?

Reply:
this is just for testing purposes, since im unable to test in. desktop anymore due websocket errors.. will roll back after its fixed